cmd/root/auth.go

```diff
@@ -152,6 +152,10 @@ func MustWorkspaceClient(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		if b := bundle.GetOrNil(cmd.Context()); b != nil {
+			err = b.InitializeWorkspaceClient()
+			if err != nil {
+				return err
+			}
 			cfg = b.WorkspaceClient().Config
```

Comment:
Then this line can use the client returned by `InitializeWorkspaceClient` instead of calling another function.

Reply:
Done